src/Catalyst.Core.Modules.Web3/ApiModule.cs

```diff
@@ -87,25 +94,34 @@ async void BuildCallback(IContainer container)
             base.Load(builder);
         }
 
-        public void ConfigureServices(IServiceCollection services, ContainerBuilder containerBuilder)
+        public void ConfigureContainer(ContainerBuilder builder)
+        {
```

Comment:
Need to find a way to use the ContainerBuilder that was DI into the Module instead of the one created with AutofacServiceProviderFactory.